Client/Frontend/Fakespot/FakespotViewController.swift

```diff
@@ -156,6 +152,25 @@ class FakespotViewController:
         viewModel.isSwiping = true
     }
 
+    func update(viewModel: FakespotViewModel, triggerFetch: Bool = true) {
+        // Only update the model if the shopping product changed to avoid unnecessary API calls
+        guard self.viewModel.shoppingProduct != viewModel.shoppingProduct else { return }
+
+        self.viewModel = viewModel
```

Comment:
By MVVM standards the view model listens to state change and updates the view. I not sure how great is to replace the whole viewModel.

Reply:
I wouldn't worry about it too much as long as it's working, we don't really use MVVM correctly anywhere in the project and it's fairly inconsistent too. The intention going forward is to use Redux instead so it's probably not worth putting effort into cleaning up how we define and use MVVM. The only reason we aren't using Redux in Fakespot at the moment is it wasn't ready yet. Just my 2 cents on the topic.

Reply:
Yup, had a discussion with Winnie about it and I don't have a strong opinion about it. Looking forward to Redux.